src/openapi/streaming/connection/transport/websocket-transport.js

```diff
@@ -360,29 +360,29 @@ WebsocketTransport.prototype.getAuthorizePromise = function(
         return Promise.reject(new Error(errorMessage));
     }
 
-    this.authorizePromise = new Promise((resolve, reject) => {
-        const options = {
-            headers: {
-                'X-Request-Id': getRequestId(),
-                Authorization: authToken,
-            },
-        };
-        const url = `${this.authorizeUrl}?contextId=${contextId}`;
+    const options = {
+        headers: {
+            'X-Request-Id': getRequestId(),

```

Comment:
Do we only use this header for batching ?
or it also serve the purpose of tracking request ?
request id sent by openapi is different than this one ?

Reply:
its just for tracking whether a request is a duplicate. I haven't changed this.